.gitignore

```diff
@@ -272,4 +272,7 @@ $RECYCLE.BIN/
 
 # Custom rules (everything added below won't be overriden by 'Generate .gitignore File' if you use 'Update' option)
 
-tmp/
\ No newline at end of file
+tmp/
+
+# Local configuration file (uncomment if needed)
+local.yaml
```

Comment:
local.yaml is supposed to be checked in when a template is used, but for the template repo we should have it as an example file (like we do with example .envs)

Reply:
Will remove this than and merge.